### ISSUE TITLE
Close on select not working when upgrading from 2.5 to 2.6

### ISSUE THREAD

**Issue by eboye** [iendeavor/vue-next-select#219]

**Comment by iendeavor** [comment]
@eboye It works for me. Could you please provide more details or a reproduce link?

I have create the following sandbox for this issue, you could fork it.

**To Reproduce**
A reproduce link:
https://codesandbox.io/s/vue-next-select-issue-219-ppizx?file=/src/views/BasicUsage.vue

Steps to reproduce the behavior:
1. Go to Basic Usage.
2. Click on select.
3. Click on first option.

**Expected behavior**
Select should be closed.

**Comment by eboye** [comment]
Hmm, I kind of fixed it ... I was not using CSS from the package but replicated it in my own scss so it could be styled with the design of the application.

Have you altered your css and possibly the way the dropdown is shown/hidden?

It would be helpful if you could provide this kind of "breaking changes" in the release notes.

**Comment by iendeavor** [comment]
Thank you for your reply.

I think it is caused by [this commit](https://github.com/iendeavor/vue-next-select/commit/16c37244a56a85124979bc326ed692b461a8931b).

But I need to consider whether I should revert it and whether it is a BREAKING CHANGE 🤔 .

**Comment by eboye** [comment]
Yup, that's the one :D

I've added this to make it work
![image](https://user-images.githubusercontent.com/624357/122090869-8157b080-ce08-11eb-94ac-a6a177ed6b19.png)

As I'm using tailwind in my project I've extracted all the css from all (not many) libraries to the scss so the css footprint could be lower.

**Comment by iendeavor** [comment]
Are the datasets and a11y attributes sufficient for all of your use case?

BTW, I would recommend you to change `&[data-is-focusing='false']` to `&[aria-expanded='true']` since the second one is more formal, and I will remove some dataset (if it can also be detect by aria-* attributes) in next major version too.

**Comment by eboye** [comment]
Yup they are, I just needed something to easily style the select and have it work with keyboard navigation and not do everything from scratch as I'm using tailwind and very little plugins ...

Sure, I'll switch to aria attrs. Good point.
